docroot/composer.json

```diff
@@ -1,58 +1,41 @@
 {
-    "name": "drupal/drupal",
-    "description": "Drupal is an open source content management platform powering millions of websites and applications.",
+    "name": "drupal/legacy-project",
```

Comment:
hmmm

Reply:
So, this one is not a "composer based" Drupal project. However, part of the notes I read somewhere said they were going to help non-composer based projects transition. They have the "recommended project" and the "legacy project." All I did was run a `drush up` and this was just one of the changes that was made by the update.